src/main/kotlin/org/sonar/fakeplugin/listener/LiveFindingsListener.kt

```diff
@@ -0,0 +1,13 @@
+package org.sonar.fakeplugin.listener
+
+import com.intellij.util.messages.Topic
+
+fun interface LiveFindingsListener {
+
+    companion object {
+        val LIVE_FINDINGS_TOPIC: Topic<LiveFindingsListener> = Topic.create("Live findings", LiveFindingsListener::class.java)
+    }
+
+    fun update(amount: String)
```

Comment:
It is strange to have a variable named `amount` of type `String`.

Reply:
Right, I forgot to change it back to Int (before I was setting some String in case of error - purely because it was more convenient and faster)